### DISCUSSION TITLE
Can this be implemented to search private databases? Or is it just an alturnate to IMDB API client lookups?

### DISCUSSION THREAD

**Discussion by protocolprowler** [Purfview/IMDb-Scout-Mod#68] [Q&A] [answered]
This project was suggest to me by user. He suggested this project could somehow enhance our projects internal search system. As time allows please reach out and let me know more details on how you think this project can integrate with ours. 

Kindest regards.

**Comment by Purfview** [comment, marked as answer]
It's not to enhance some external projects, it's a userscript, ready to use for the users, external "projects" can be linked in it so users can find where content is or not from the IMDb site, for example find content in subtitle sites, streams, amazon, swarmazon or your plex server, ect., and other functions related to the movies/series.
Something like Jackett, but more comfortable as it works from your internet browser.
